modules/aggregation-layers/src/utils/cpu-aggregator.js

```diff
@@ -41,6 +41,16 @@ const defaultDimensions = [
         },
         aggregation: {
           prop: 'colorAggregation'
+        },
```

Comment:
`getBins` dimension is handling both `Filtering` and `Sorting`, can we split into two dimensions, `getFilteredBins` and `getSortedBins`? This will be inline with 8.0 refactor of aggregation layers as-well. Not all layers going to use filtering.

Reply:
actually , we can make sorting optional, which is more simpler change, I can do that in master during the refactor, this is ok for 7.3.